config/locales/client.en.yml

```
@@ -223,6 +223,7 @@ en:
           copy: "Copy"
           copied: "Copied!"
           cancel: "Cancel"
+          insert_footnote: "Insert as footnote"
```

```suggestion
          insert_footnote: "Add footnote"
```

Insert sounds odd to me, maybe it's just me. It is technically correct, but "add" is fewer characters and feels less technical.